docs/pages/deploy/build-project.mdx

```diff
@@ -1,28 +1,124 @@
+  <Tab>
+    <Terminal cmd={['$ eas build --platform android']} />
+  </Tab>
+  <Tab>
+    <Terminal cmd={['$ eas build --platform ios']} />
+  </Tab>
+</Tabs>
+
+> You can attach a message to the build by passing `--message` to the build command, for example, `eas build --platform ios --message "Some message"`. The message will appear on the website. It comes in handy when you want to leave a note with the purpose of the build for your team.
+> Alternatively, you can use `--platform all` option to build for Android and iOS at the same time:
+
+<Terminal cmd={['$ eas build --platform all']} />
+
+## Developer account
+
+You will need to have a developer account for the app store you want to submit your app.
+
+<Collapsible summary="Google Play Developer membership is required to distribute to the Google Play Store.">
+
+You can build and sign your app using EAS Build, but you can't upload it to the Google Play Store unless you have a membership, a one-time $25 USD fee.
+
+</Collapsible>
+
+<Collapsible summary="Apple Developer Program membership is required to build for the Apple App Store.">
+
+If you are going to use EAS Build to create release builds for the Apple App Store, you need access to an account with a $99 USD [Apple Developer Program](https://developer.apple.com/programs) membership.
+
+</Collapsible>
+
+## App signing credentials
+
+Before the build process can start for app stores, you will need to have a store developer account and generate or provide app signing credentials.
+
+Whether you have experience with generating app signing credentials or not, EAS CLI does the heavy lifting. You can opt-in for EAS CLI to handle the app signing credentials process.
+
+### Android app signing credentials
+
+- If you have not yet generated a keystore for your app, you can let EAS CLI take care of that for you by selecting `Generate new keystore`, and then you are done. The keystore is stored securely on EAS servers.
+- If you want to manually generate your keystore, see the [manual Android credentials guide](/app-signing/local-credentials#android-credentials) for more information.
+
+### iOS app signing credentials
+
+- If you have not generated a provisioning profile and/or distribution certificate yet, you can let EAS CLI take care of that for you by signing in to your Apple Developer Program account and following the prompts.
+- If you want to manually generate your credentials, see the [manual iOS credentials guide](/app-signing/local-credentials#ios-credentials) for more information.
+
+## Wait for the build to complete
+
+By default, the `eas build` command will wait for your build to complete, but you can interrupt it if you prefer not to wait. Monitor the progress and read the logs by following the link to the build details page that EAS CLI prompts once the build process gets started. You can also find this page by visiting [your build dashboard](https://expo.dev/builds) or running the following command:
```

Comment:
```suggestion
By default, the `eas build` command will wait for your build to complete, but you can interrupt it if you prefer not to wait. Instead, use the builds details page link prompted by EAS CLI to monitor the build progress and read the build logs. You can also find this page by visiting [your build dashboard](https://expo.dev/builds) or running the following command:
```

Let's improve the readability of the second sentence. There's no need to mention "build process gets started". We are already doing that in the first sentence.